architecture/core_concepts/routes.adoc

```diff
@@ -220,18 +225,19 @@ implementing stick-tables that synchronize between a set of peers.
 
 Specific configuration for this router implementation is stored in the
 *_haproxy-config.template_* file located in the *_/var/lib/haproxy/conf_*
-directory of the router container.
+directory of the router container. The file may be
+xref:../../install_config/router/customized_haproxy_router.adoc#[customized].
 
 [NOTE]
 ====
-The `balance source` directive does not distinguish between external client IP
+The `source` load balancing strategy does not distinguish between external client IP
 addresses; because of the NAT configuration, the originating IP address
 (HAProxy remote) is the same. Unless the HAProxy router is running with
 `*hostNetwork: true*`, all external clients will be routed to a single pod.
 ====
 
 [[env-variables]]
-== Configuration Parameters
+== Configuration Parameters - Environment Variables
```

Comment:
Feels clunky... @bfallonf can you think of a better name?

Configuration Variables
Configuration Environment Variables

Dunno...

Also, it would be good to link to the command line config too for the router (or just move this section there in the future)